src/components/Buyer.tsx

```diff
@@ -24,13 +25,15 @@ const Buyer = ({
   const { id } = route.params
   const buyer = buyers.find((buyer) => buyer.id === id)
   if (!buyer) return navigation.back()
+
+  useEffect(() => {
+    navigation.setOptions({ title: buyer.name })
+  }, [])
```

Comment:
Jag tror att buyer måste finnas med här för att den ska köra koden när köparen ändras.
```suggestion
  }, [buyer])
```